src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallDao {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
According to the checklist, the `get()` method in DAO should return `Optional<CinemaHall>` instead of `CinemaHall` directly. This is to properly handle cases where a `CinemaHall` with the given ID does not exist.